Makefile.work

```diff
@@ -477,6 +477,7 @@ SONIC_SLAVE_BASE_BUILD = \
 		$(DOCKER_SLAVE_BASE_PULL_REGISTRY); \
 	} || \
 	{ \
+		set -o pipefail ; \
```

Comment:
'ls non-exist-file | grep xxx'      It will not run grep command. This is pipefail.
`
cat a.sh 
echo HH && { set -o pipefail; echo AA; ls oieaofijefojaoij; echo BB; }
bash a.sh
HH
AA
ls: cannot access 'oieaofijefojaoij': No such file or directory
BB
`